topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -131,29 +127,253 @@ Phred Quality Score | Probability of incorrect base call | Base call accuracy
 {: .question}
 
 > ### {% icon comment %} Comment
-> The current lllumina (1.8+) uses Sanger format (Phred+33). If you are working with older datasets you may encounter the older scoring schemes. **FastQC** {% icon tool %}, the tool we will use in the next step, can be used to try to determine what type of quality encoding is used (through assessing the range of Phred values seen in the FASTQ).
+> The current lllumina (1.8+) uses Sanger format (Phred+33). If you are working with older datasets you may encounter the older scoring schemes. **FastQC** {% icon tool %}, a tool we will use later in this tutorial, can be used to try to determine what type of quality encoding is used (through assessing the range of Phred values seen in the FASTQ).
 {: .comment}
 
-When looking at the file in Galaxy, it looks like most the nucleotides have a high score (`I` corresponding to a score 40). Is it true for all sequences? And along the full sequence length?
+When looking at the file in Galaxy, it looks like most the nucleotides have a high score (`G` corresponding to a score 38). Is it true for all sequences? And along the full sequence length?
+
+## Assess quality with FASTQE 🧬😎
+
+To estimate sequence quality along all sequences, we now use [FASTQE](https://fastqe.com/). It is an open-source tool that provides a simple way to quality control raw sequence data and print them as emoji. You can use it to give a quick impression of whether your data has any problems of which you should be aware before doing any further analysis.
+
+> ### {% icon hands_on %} Hands-on: Quality check
+>
+> 1. {% tool [FASTQE](toolshed.g2.bx.psu.edu/repos/iuc/fastqe/fastqe/0.2.6+galaxy2) %} with the following parameters
+>    - {% icon param-files %} *"FastQ data"*: `female_oral2.fastq-4143.gz`
+>    - {% icon param-select %} *"Score types to show"*: `Mean`
+>
+> 2. Inspect the generated HTML file
+>
+{: .hands_on}
+
+Rather than looking at quality scores for each individual read, FASTQE looks at quality collectively across all reads within a sample and can calculate the mean for each nucleotide position along the length of the reads. Below shows the mean values for this dataset.
+
+![FASTQE before](../../images/quality-control/fastqe-mean-before.png "FASTQE mean scores")
+
+You can see the score for each emoji [here](https://github.com/fastqe/fastqe#scale). The emojis below, with Phred scores less than 20, are the ones we hope we don't see much.
+
+Phred Quality Score | ASCII code | Emoji
+--- | --- | ---
+0 | ! | 🚫
+1 | " | ❌
+2 | # | 👺
+3 | $ | 💔
+4 | % | 🙅
+5 | & | 👾
+6 | ' | 👿
+7 | ( | 💀
+8 | ) | 👻
+9 | * | 🙈
+10 | + | 🙉
+11 | , | 🙊
+12 | - | 🐵
+13 | . | 😿
+14 | / | 😾
+15 | 0 | 🙀
+16 | 1 | 💣
+17 | 2 | 🔥
+18 | 3 | 😡
+19 | 4 | 💩
+
+
+> ### {% icon question %} Questions
+>
+> What is the lowest mean score in this dataset? 
+>
+> > ### {% icon solution %} Solution
+> > The lowest score in this dataset is 😿 13.
+> {: .solution }
+{: .question}
+
+
+## Improve quality with Cutadapt
```

Comment:
I love that you added a dataset that shows a clearer improvement before and after QC! 

Would it maybe make sense to show FastQC before Cutadapt as well? and then after cutadapt participants can observe the before/after improvements with both tools?

Reply:
Thanks @shiltemann ! I wasn't expecting a review during GCC I just wanted to finish this off (finally). 

>Would it maybe make sense to show FastQC before Cutadapt as well? and then after cutadapt participants can observe the before/after improvements with both tools?

It could do. I was afraid to make it too long but I'll see how I could fit it in, maybe people can just skip bits if they don't want to do/see both.

Reply:
yeah fair point, maybe we can just amend the FastQC section to also do the before & after, and give users a choice of tool orso, not sure. But yes, things that can wait until after GCC (or maybe a CoFest activity?). Thanks for finishing it!

Reply:
Ok I've added it as a CoFest activity :)
Just noting here, I was considering adding in fastp while I was updating this tut as it's used in other tutorials. Thought it might make sense to show it in this QC tutorial as an optional step. But unlike FastQC it didn't detect the adapter, not sure why so I've that out for the moment.